### PR TITLE
Add meta tag if it doesn't exist

### PR DESCRIPTION
**Pull request by Daniel-Khodabakhsh**
###  Summary

Add <meta> tag if it doesn't exist. Feature request in #12 

### Requirements

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackhq/csp-html-webpack-plugin/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
* [x] I've written tests to cover the new code and functionality included in this PR.
* [x] I've read, agree to, and signed the [Contributor License Agreement (CLA)](https://cla-assistant.io/slackhq/csp-html-webpack-plugin).

### CODE REVIEW

File: spec/plugin.spec.js [hunk 1]
```diff
@@ -286,6 +286,45 @@ describe('CspHtmlWebpackPlugin', () => {
     );
   });
 
+  it('adds meta tag when no template is specified', done => {
```
Comment (AnujRNair) [inline spec/plugin.spec.js]: Maybe `adds meta tag with completed policy when the entire meta tag is missing`?

File: spec/plugin.spec.js [hunk 2]
```diff
@@ -286,6 +286,45 @@ describe('CspHtmlWebpackPlugin', () => {
...
+    const webpackConfig = {
+      entry: path.join(__dirname, 'fixtures/index.js'),
+      output: {
+        path: OUTPUT_DIR,
+        filename: 'index.bundle.js'
+      },
+      plugins: [
+        new HtmlWebpackPlugin({
+          filename: path.join(OUTPUT_DIR, 'index.html'),
+          inject: 'body'
```
Comment (AnujRNair) [inline spec/plugin.spec.js]: Would you be able to create a new template `with-no-meta-tag.html` and explicitly set it here, using the `template` param? Example above :) 

That way, if other examples change, this one won't start failing

Comment (Daniel-Khodabakhsh) [reply spec/plugin.spec.js]: Okay I'll add a `with-no-meta-tag.html` test case.

Wouldn't you still want this original test case in addition to what you requested? This test case specifies no template at all, using the automatically generated template from HtmlWebpackPlugin. How would it start failing if other examples changed?

Comment (AnujRNair) [reply spec/plugin.spec.js]: No template defaults to an index.html file, so if your changes are able to add the missing meta tag to the specified template here, it should add it to the one with no template too!

I was initially thinking that if HTML Webpack Plugin changed something (like started requiring a template), then this example may start failing

**Pull request merged**